src/App.jsx

```diff
@@ -21,7 +22,12 @@ function App() {
 			<h1>NC-News</h1>
 			<Navbar />
 			<Routes>
-				<Route path="/" element={<Articles />} />
+				<Route
+					path="/"
+					element={<Navigate to="/articles" replace={true} />}
+				></Route>
+				<Route path="/articles" element={<Articles />} />
+				<Route path="/articles/:topic" element={<Articles />} />
 				<Route
 					path="/article"
 					element={<ArticleContents currentUser={currentUser} />}
```

Comment:
Do we need a topics route as below? We don't have a topics page, only articles that can take a parameter to help us filter

Reply:
I think I'm going to refactor the topic drop down button that I'm currently using to some kind of autocomplete search but it's low down my prio list. Due to this I wanted a page to add and see all the topics that are available.